packages/stencil/src/components/datacom-display-card/datacom-display-card.tsx

```diff
@@ -0,0 +1,42 @@
+import { Component, Host, h, Prop } from '@stencil/core';
+
+/**
+ *Display card component is used to display text image and navigation together using a clear visual hierarchy.
+ */
+
+@Component({
+  tag: 'datacom-display-card',
+  styleUrl: 'datacom-display-card.css',
+  shadow: true,
+})
+export class DatacomDisplayCard {
+  @Prop() url?: string;
+  @Prop() heading: string;
+  @Prop() ctaText: string;
+  @Prop() imageUrl: string;
+
+  render() {
+    return (
+      <Host>
+        <div class="dc-display-card-wrapper" tabIndex={1}>
```

Comment:
Probably best to use a tabindex of 0 here instead of a positive number. You might mess with the tab order when using keyboard navigation. (See below warning from the [Mozilla site](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/tabindex))

![image](https://user-images.githubusercontent.com/38770596/223550183-47125b83-f339-43c9-8141-972b21f66111.png)